picard/tagger.py

```diff
@@ -1110,9 +1138,15 @@ def main(localedir=None, autoupdate=True):
         picard_args.stand_alone_instance,
     }
 
+    to_be_added = []
+    for x in picard_args.FILE_OR_URL:
+        if not urlparse(x).netloc:
+            x = os.path.abspath(x)
+        to_be_added.append(x)
+
     if not should_start:
         try:
-            pipe_handler = pipe.Pipe(app_name=PICARD_APP_NAME, app_version=PICARD_FANCY_VERSION_STR, args=[os.path.abspath(x) for x in picard_args.FILE])
+            pipe_handler = pipe.Pipe(app_name=PICARD_APP_NAME, app_version=PICARD_FANCY_VERSION_STR, args=to_be_added)
```

Comment:
`to_be_added` is only used here. We should move the preprocessing code above here too.